Dockerfile

```diff
@@ -4,10 +4,7 @@ MAINTAINER Team Aruha, team-aruha@zalando.de
 
 WORKDIR /
 ADD build/libs/nakadi.jar nakadi.jar
-ADD plugins plugins/
-ADD scm-source.json /scm-source.json
 
 EXPOSE 8080
 
-# run the server when a container based on this image is being run
-ENTRYPOINT java $JAVA_OPTS -Dloader.path=/plugins -Djava.security.egd=file:/dev/./urandom -jar nakadi.jar
\ No newline at end of file
+ENTRYPOINT java -jar nakadi.jar
```

Comment:
@v1ctor On local machine it was started up just fine. I left it for private part. In case you will experience issues with that we can add it in the future.